Formula/fits.rb

```diff
@@ -35,8 +35,10 @@ def install
     # fits-env.sh is a helper script that sets up environment
     # variables, so we want to tuck this away in libexec
     libexec.install "fits-env.sh"
-    bin.install "fits.sh" => "fits"
-    bin.install "fits-ngserver.sh" => "fits-ngserver"
+    bin.install "fits.sh"
```

Comment:
Something I forgot; this line and the one below can be combined as `bin.install "fits.sh", "fits-ngserver.sh"`.